internal/pkg/addon/storage.go

```diff
@@ -15,11 +15,19 @@ import (
 )
 
 const (
-	dynamoDbTemplatePath = "addons/ddb/cf.yml"
-	s3TemplatePath       = "addons/s3/cf.yml"
-	rdsTemplatePath      = "addons/aurora/cf.yml"
-	rdsRDWSTemplatePath  = "addons/aurora/rdws/cf.yml"
-	rdsRDWSParamsPath    = "addons/aurora/rdws/addons.parameters.yml"
+	dynamoDbTemplatePath  = "addons/ddb/cf.yml"
+	s3TemplatePath        = "addons/s3/cf.yml"
+	rdsTemplatePath       = "addons/aurora/cf.yml"
+	rdsV2TemplatePath     = "addons/aurora/serverlessv2.yml"
+	rdsRDWSTemplatePath   = "addons/aurora/rdws/cf.yml"
+	rdsRDWSV2TemplatePath = "addons/aurora/rdws/serverlessv2.yml"
+	rdsRDWSParamsPath     = "addons/aurora/rdws/addons.parameters.yml"
+)
+
+const (
+	// Aurora Serverless versions.
+	ServerlessVersionV1 = "v1"
+	ServerlessVersionV2 = "v2"
```

Comment:
nit: do these need to be public?
```suggestion
	auroraServerlessVersionV1 = "v1"
	auraraServerlessVersionV2 = "v2"
```

Reply:
`addon.AuroraServerlessVersion` is used in `newRDSTemplate()` of `cli/storage_init.go`. This function calls `addon.NewServerlessVxTemplate` using switch statement, same as `addon.RDSEngineTypeMySQL` or `addon.RDSEngineTypePostgreSQL`.